torchtext/utils.py

```diff
@@ -25,7 +27,18 @@ def inner(b=1, bsize=1, tsize=None):
 
 
 def download_from_url(url, path):
-    """Download file, with logic (from tensor2tensor) for Google Drive"""
+    """Download file, with logic (from tensor2tensor) for Google Drive
```

Comment:
We could take this as a chance to move this stuff into pytorch core and then use it here.

Reply:
yeah. As long as they are not text specific (I think most of them are not), we can move them to the core library.